web/app/settings/settings.screensaver.html

```diff
@@ -0,0 +1,192 @@
+                            <select id="ss_onStopType" class="d-block" name='onStopType' ng-model="config.onStop.type" ng-change="validate()">
+                                <option translate-keep-content translate="screensaver.settings.common.stop" value='stop'>Stop</option>
+                                <option translate-keep-content translate="screensaver.settings.common.gotodashboard" value='gotodashboard'>Go to dashboard</option>
+                            </select>
+                            <span class='helpinfo'>
+                                <a ng-click='onStopTypeHelp = !onStopTypeHelp'>
+                                    <em>
+                                        <span translate-keep-content translate="screensaver.settings.onstop.info">What to do when screensaver starts</span>
+                                        <span class='text-info glyphicon glyphicon-question-sign'></span>
+                                    </em>
+                                </a>
+                                <ul ng-if="onStopTypeHelp">
+                                    <li translate-keep-content translate="screensaver.settings.onstoptype.help.line1">Stop - simply stops screensaver wherever it may be.</li>
+                                    <li translate-keep-content translate="screensaver.settings.onstoptype.help.line2">Go to dashboard - activates the selected dashboard.</li>
+                                </ul>
+                            </span>
+                        </div>
+                    </li>
+                    <section ng-if="config.onStop.type!=='stop'">
+                        <li>
+                            <label>
+                                <em translate-keep-content translate="screensaver.settings.common.dashboard">Dashboard:</em>
+                            </label>
+                            <select class="d-block" name='dashboard' ng-model="config.onStop.dashboard" ng-change="validate()" ng-init="config.onStop.dashboard = config.onStop.dashboard ? config.onStop.dashboard : dashboards[0].id">
+                                <option ng-repeat="db in dashboards | orderBy: db.id" value="{{db.id}}">{{db.id}}</option>
+                            </select>
+                        </li>
+                    </section>
+                </ul>
+            </div>
+            <div class="scrollable box" ng-init="showAdvancedSettings=false">
+                <div>
+                    <h4 class="inline" translate="{{::translations.headingadvanced}}"></h4>
+                    <label ng-click="showAdvancedSettings=!showAdvancedSettings">
+                        <input type="checkbox" ng-model="showAdvancedSettings" id="showAdvancedSettings" ng-change="showAdvancedSettings=!showAdvancedSettings">
+                        <a>{{::translations.showadvanced}}</a>
+                    </label>
+                </div>
+                <ul class="settings-outer" ng-if="showAdvancedSettings">
```

Comment:
I'm actually wondering if all that customization really adds value for the end user. Sensible hardcoded defaults could be enough, with an eventual textbox with csv values to allow events to be customized if deemed necessary for the (really) advanced users.